airflow/providers/amazon/aws/waiters/emr.json

```diff
@@ -94,6 +94,37 @@
                     "state": "failure"
                 }
             ]
+        },
+        "job_step_wait_for_terminal": {
```

Comment:
This is a nit, but `job_step` sounds odd. There is a native waiter for EMR steps called `step_completed` (rather than `job_step_completed`). Maybe this one can be named `step_terminal_state` or something along that line?

Reply:
ok, changed